packages/rlc-codegen/src/helpers/camelCase.ts

```diff
@@ -0,0 +1,14 @@
+export function camelCase(
+  str: string,
```

Comment:
We have three folders: 

- a) autorest.typescript
- b) cadl-rlc-emitter 
- c) rlc-codegen.

Just want to confirm one thing: the common part which will be used in a) + b) will be defined in c), Right?

Reply:
Your understanding is correct. Also I agree we'll need to rename our folders and packages. I think I'll do that in a PR of itself as a follow-up. This is my proposal

`cadl-rlc-emitter` => `cadl-typescript` (@azure-tools/cadl-typescript)
`rlc-codegen` => `typescript-codegen-common` (@azure-tools/typescript-codegen-common)